### PR TITLE
Feature/impove login seo #455

### PR DESCRIPTION
**Pull request by MOHIT-PANDEY215**
#455 Fix: Improve SEO on login page.
image

### CODE REVIEW

File: apps/frontend/src/routes/login/index.tsx [hunk 1]
```diff
@@ -97,3 +97,48 @@ export default component$(() => {
...
+  title: 'Reduced.to | Login',
+  meta: [
+    {
+      name: 'title',
+      content: 'Reduced.to | Login',
+    },
+    {
+      name: 'description',
+      content:
+        'Login to your Reduced.to account to manage your links and QR-Codes.',
```
Comment (origranot) [inline apps/frontend/src/routes/login/index.tsx:111]: We are not managing QR-Codes in our system.
Change the description to be something like: "Login to your Reduced.to account to manage your shortened links"

**Review by origranot — CHANGES_REQUESTED** [review]
@MOHIT-PANDEY215 Looks good 🚀 

There is a small comment I left, would you mind resolve it?
Also, please run the formatter on your changes using: `npx nx format:write`

**Comment by MOHIT-PANDEY215** [comment]
@origranot Thank you for the comment. I have applied the changes.

**Comment by origranot** [comment]
@MOHIT-PANDEY215 Hey,
can you please run the formatter on your changes?

**Pull request merged**